CONTRIBUTING.rst

```diff
@@ -10,17 +10,17 @@ These are some contribution examples
 
 - Extending the documentation.
 
-- Sponsor a Sprint (https://plone.org/events/sprints/whatis).
+- Sponsor a `sprint <https://plone.org/news-and-events/events/sprints>`_.
 
 
 For pull requests, keep this in mind
 ------------------------------------
 
 - Add a test which proves your fix and make it pass.
 
-- Describe your change in ``CHANGES.rst``
```

Comment:
I wonder if we could instead or additionally link. We have links to headings included.
What is your preference?

Reply:
I don't understand your question. Also this file should have all its content removed and point to `docs/contributing.rst`, which is now the authoritative source. I'll resolve this comment, but please reopen or create a new issue if it hasn't already been resolved.